src/multicore.rs

```diff
@@ -65,6 +65,19 @@ impl Worker {
 
         THREAD_POOL.scoped(|scope| f(scope, chunk_size))
     }
+
+    pub fn scoped<'a, F, R>(&self, f: F) -> R
```

Comment:
Some documentation would be good (especially since `scope` and `scoped` are so similar in their names).

Reply:
done